### ISSUE TITLE
Fuzz serialization inputs

### ISSUE THREAD

**Issue by ben-chain** [plasma-group/plasma-utils#37]
**Is your feature request related to a problem? Please describe.**
Right now, we're using hardcoded objects for all of the serialization testing in `test/serialization`.  

**Describe the solution you'd like**
Ideally, we would also decode fuzzed inputs for better coverage, where the encodings are generated by an RNG.  It would generate random strings which are still well formed (this might mean having valid sigs, start > end, and the right byte for the `numElements` in array encodings) and then check decoding passes

**Comment by kfichter** [comment]
Good idea, might as well also try invalid inputs and check that decoding fails.

**Comment by kfichter** [comment]
Actually now that I think about it, fuzz testing is probably something that shouldn’t be part of the normal test suite. When RNG is involved we typically want repeatability - that means setting a standard seed. Unfortunately that also defeats the purpose of fuzz testing! 

Might be better to throw a file in /tests that allows us to run fuzz tests in the background while we’re doing something else and just have it spit out an error + offending inputs when something goes wrong.
